### ISSUE TITLE
Update peerDependencies to be aware of React 15.0.0

### ISSUE THREAD

**Issue by mtoso** [akiran/react-slick#262]
I am trying to update the code base where I am using `react-slick` to use react@15.0.0-rc.1 react-dom@15.0.0-rc.1. I am using npm@2.11.3 so I have an error on the peerDependencies because this is the actual definition: 
```
"peerDependencies": {
    "react": "^0.13.0 || ^0.14.0"
  }
```

According to this [release note](https://facebook.github.io/react/blog/#bring-everyone-along) by Facebook most of the user (79%) of React are on the 0.14 version. 

What about deprecate react 0.13 and move the peerDependencies to:
```
"peerDependencies": {
    "react": "^0.14.0 || ^15.0.0-rc.1"
  }
```

**Comment by Tamiyadd** [comment]
I'm having the same issue too now that react 15.0.1 is released

**Comment by akiran** [comment]
Released react-slick@0.12 with support to react@15.0
